.github/actions/package/linux/action.yml

```diff
@@ -113,7 +113,10 @@ runs:
       run: |
         cmake --install ${{ env.BUILD_DIR }} --prefix ${{ env.INSTALL_PORTABLE_DIR }}
         cmake --install ${{ env.BUILD_DIR }} --prefix ${{ env.INSTALL_PORTABLE_DIR }} --component portable
-
+        if [ $APPIMAGE_ARCH == 'x86_64' ]; then
```

Comment:
This is also probably an issue on other architectures

So if we do end up doing this, I think it'd be a better choice to add this to our CMake install scripts - as that can better handle discovery of bzip2 and make sure this workaround always implies with `--component portable`